src/ng-xform/ng-xform.component.ts

```diff
@@ -72,7 +72,7 @@ export class NgXformComponent implements OnInit, OnChanges {
   private getAttributeValue(attr: string, value: any): any {
```

Comment:
Didn't you removed this method? It is in another branch?

Reply:
yep